pkg/internalapis/cnsoperator/config/triggercsifullsync_crd.yaml

```diff
@@ -10,63 +10,62 @@ spec:
-            error:
-              description: The last error encountered during import operation, if any.
-              type: string
-  version: v1alpha1
   versions:
   - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: TriggerCsiFullSync is the Schema for the triggercsifullsyncs API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            type: object
+            description: TriggerCsiFullSyncSpec is the spec for TriggerCsiFullSync
+            properties:
+              triggerSyncID:
+                description: TriggerSync gives an option to trigger full sync on demand.
+                type: integer
+            required:
+            - triggerSyncID
+          status:
+            type: object
+            description: TriggerCsiFullSyncStatus defines the observed state of TriggerCsiFullSync
+            properties:
+              inProgress:
+                description: InProgress indicates whether a CSI full sync is in progress.
+                type: boolean
+              lastTriggerSyncID:
+                description: LastTriggerSyncID indicates the last trigger sync Id.
+                type: integer
+              lastSuccessfulStartTimeStamp:
+                description: LastSuccessfulStartTimeStamp indicates last successful full sync start timestamp.
+                format: date-time
+                type: string
+              lastSuccessfulEndTimeStamp:
+                description: LastSuccessfulEndTimeStamp indicates last successful full sync end timestamp.
+                format: date-time
+                type: string
+              lastRunStartTimeStamp:
+                description: LastRunStartTimeStamp indicates last run full sync start timestamp.
+                format: date-time
+                type: string
+              lastRunEndTimeStamp:
+                description: LastRunEndTimeStamp indicates last run full sync end timestamp.
+                format: date-time
+                type: string
+              error:
+                description: The last error encountered during import operation, if any.
+                type: string
     served: true
```

Comment:
schema. openAPIV3Schema.type is a required field and must be set without which CRD will fail to create.
Please set `type: object` under openAPIV3Schema.